src/EnergyPlus/DataSurfaces.cc

```diff
@@ -251,7 +251,7 @@ Real64 SurfaceData::getInsideAirTemperature(EnergyPlusData &state, const int t_S
         auto &inletNodes = (state.dataHeatBal->doSpaceHeatBalance) ? state.dataZoneEquip->spaceEquipConfig(this->spaceNum).InletNode
                                                                    : state.dataZoneEquip->ZoneEquipConfig(Zone).InletNode;
         for (int nodeNum : inletNodes) {
-            auto &inNode = state.dataLoopNodes->Node(state.dataZoneEquip->ZoneEquipConfig(Zone).InletNode(nodeNum));
+            auto &inNode = state.dataLoopNodes->Node(nodeNum);
```

Comment:
Oh dear, yeah, I missed that entirely.

Reply:
In the unit test that covers this, there just happens to be 4 inlet nodes, and the node numbers are 1-4, so it worked. I'll try to remember to improve that unit test (all of the inlet node conditions are the same currently, so it could be a better test).